pelicanconf.py

```diff
@@ -25,7 +25,7 @@
 	# ('Meetup', 'https://www.meetup.com/BAyPIGgies/'),
          ('Python.org', 'http://python.org/'),
          ('Mailing List', 'https://mail.python.org/mailman/listinfo/baypiggies'),
-         # ('You can modify those links in your config file', '#')
+         ('Code of Conduct', '/code_of_conduct.html'),
```

Comment:
Since we organize directory structures for our meetings, book reviews, and job_postings, does it make sense that we also have a URL pattern that matches that organization (e.g. `/community/code_of_conduct.html`)?

I left this just as the slug as that's what we seem to be doing with the meetings.

Reply:
Also, do we want to make a new section for this instead of just adding an extra link to the link section?